mulighetsrommet-tiltaksokonomi/src/main/kotlin/no/nav/tiltak/okonomi/service/OkonomiService.kt

```diff
@@ -119,7 +117,7 @@ class OkonomiService(
             }
             .map {
                 log.info("Lagrer bestilling ${bestilling.bestillingsnummer} som annullert")
-                queries.bestilling.setStatus(bestillingsnummer, BestillingStatusType.ANNULLERT)
+                queries.bestilling.setStatus(bestillingsnummer, BestillingStatusType.SENDT)
```

Comment:
Melding og status matcher ikke helt her. Setter vi til SENDT-status fordi vi har sendt annulleringen, men venter svar? Burde kanskje vært ANNULLERING_SENDT i stedet slik at vi ikke mikser med SENDT som settes ifm opprett-bestilling?

Reply:
(Evt. endre routes tilbake til strings i stedet for `@Resource("<route>")`, jeg tok det i bruk her for å evt. danne noen erfaringer på om det er nice eller ei, i tilfelle det er noe vi kunne ønsket å ta i bruk i api også)

Reply:
Ja ANNULLERING_SENDT er nok lurt